.github/workflows/dask_test.yaml

```
@@ -47,7 +47,7 @@ jobs:
           cache-environment-key: environment-${{ steps.date.outputs.date }}-0
 
       - name: Install current main versions of dask
-        run: python -m pip install git+https://github.com/dask/dask
+        run: python -m pip install git+https://github.com/phofl/dask@legacy-dataframe
```

TODO revert before merging

```suggestion
        run: python -m pip install git+https://github.com/dask/dask
```